distribution/packages/src/common/scripts/preinst

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
```

Comment:
This should be `/bin/bash`, that's what all of our scripts require.

Reply:
If we could switch to /bin/bash it would be great to make the code more readable too e.g. conditionals etc. as now I was limited to POSIC onlyu. Defaulting to `/bin/bash` however, I fear it is dangerous, as the script is common across all platforms and there's no guarantee that /bin/bash is present on all platforms, e.g. Debian defaults to /bin/dash.

Reply:
However, our packages already depend on `bash` so we should be safe. I will commit this change ASAP. Thanks!

Reply:
Addressed in f1fd2a2